skymp5-server/cpp/server_guest_lib/Faction.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <nlohmann/json.hpp>
+#include <simdjson.h>
+
+struct Faction
+{
+  uint32_t formId;
+  int8_t rank;
+
+  friend bool operator==(const Faction& r, const Faction& l)
+  {
+    return r.formId == l.formId;
```

Comment:
r.rank == l.rank

Reply:
Why? I just want to check, whether the faction the same in its id.

Reply:
answered in dm